src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs

```diff
@@ -4062,6 +4091,16 @@ internal bool SetValueInternal(int rowIndex, object value)
             return SetValue(rowIndex, value);
         }
 
+        private bool ShowToolTip(int rowIndex)
```

Comment:
`Show` implies action, whereas here we check whether we must proceed with the action.
```suggestion
        private bool ShouldShowToolTip(int rowIndex)
```